std/uni.d

```diff
@@ -8214,6 +8215,162 @@ unittest //12428
     assert(s == "abcdefghij");
 }
 
+
+// generic toUpper/toLower on whole range, returns range
+private auto toCaser(alias indexFn, uint maxIdx, alias tableFn, Range)(Range str)
+    if (isInputRange!Range && isSomeChar!(ElementEncodingType!Range))
+{
+    static if (!is(Unqual!(ElementEncodingType!Range) : const(dchar)))
```

Comment:
No, as that picks up dchar's, too. I want to test for everything that is NOT a dchar. I want to use byDchar on anything that is not a dchar.